packages/main/src/webComponents/Switch/Switch.stories.tsx

```diff
@@ -12,9 +12,9 @@ export const generatedDefaultStory = () => (
   <Switch
     checked={boolean('checked', false)}
     disabled={boolean('disabled', false)}
-    textOn={text('textOn', 'Yes')}
-    textOff={text('textOff', 'No')}
-    graphical={boolean('graphical', false)}
+    graphical={text('graphical', false)}
```

Comment:
`graphical` expects to be a boolean.
 ```suggestion
     graphical={boolean('graphical', false)}
```